src/features/locators/components/LocatorIcon.tsx

```diff
@@ -38,7 +38,10 @@ export const LocatorIcon: React.FC<Props> = ({ message, locator, deleted }) => {
 
     switch (locator.taskStatus) {
       case locatorTaskStatus.SUCCESS: {
-        return getLocatorValidationStatus(message) === ValidationStatus.WARNING ? warningEditedIcon : null;
+        const validationStatus = getLocatorValidationStatus(message);
+        return validationStatus === ValidationStatus.WARNING || validationStatus === ValidationStatus.ERROR
```

Comment:
I think that we don't need additional comparison for error validation status, because we don't allow user to create or to edit locator with error.

Reply:
it's not for saving, it's for displaying an icon for already saved